docs/introduction/requirements.md

```diff
@@ -0,0 +1,46 @@
+# Requirements
+
+## Quantitative values
+
+### Real time tracking of resources
+
+*present*
+
+It involves `vf:currentQuantity` as well as *available quantity*. They might differ for example when ongoing *use* event(s) involves certain quantity of resource. *Available quantity* would equal `vf:currentQuantity` minus all the quantities currently involved in *use* event(s).
+
+Use cases:
+
+* Ouikit
+* Transportation
+
+### Planning schedules
+
+*future*
+
+It mostly relate to flows with *use* and *work* action, but it may also come into play for other actions. For example *load* and *unload* might require very specific scheduling. Last but not least it might play role in *service quotas*, for example grid utilities providers might need to play different volumes of *consume* in different time periods. For *consume* and *produce* precise scheduling might play greater role for resources with limited buffering. Mentioned grid utilities or perishable products give two examples. Also [*just in time manufacturing*](https://en.wikipedia.org/wiki/Just-in-time_manufacturing) would have stricter scheduling requirements.
+
+Use cases:
+
+* Ouikit
+* Harvesting
+* Metered electricity
+* Shared studios
+
+### Social accounting
+
+*past*, *future*
+
+It plays role when agents make agreements which involve reciprocity. For *use* it might combine quantity of used resource and duration of the usage. For *service* it might depend on various factors and specific Terms of Service. In some cases it might involve quantities like *number of words*, *distance* etc.
```

Comment:
The people we know who wanted to count words for translations wanted to do it on a `work` event.

Reply:
To clarify that specific use case, do they count words from the original document or words in the translated document?

Reply:
I guess in that translations UC to meet *Social accounting* requirement people doing translation work would use either *word count* or *work hours* for *accountingQuantity*. On the other hand to meet *Reports and analysis* use case they might need both factors, for example to have some averages of words per hours for different languages and domains, based on which they could do time estimates and plan their working schedules.

Reply:
I don't know, could ask if it is important.  I'm sure there is an industry standard for that.  I don't tend to think it matters, because this is words on the flow.

Reply:
I moved some citations specific conversation to related issue https://github.com/valueflows/valueflows/issues/449#issuecomment-476400049

Reply:
>On the other hand to meet Reports and analysis use case they might need both factors, for example to have some averages of words per hours for different languages and domains, based on which they could do time estimates and plan their working schedules.

Nobody in our experience has needed both.  We could run into it in the future of course, but I'd prefer to stick with known use cases as much as possible.